src/components/app/details/cicdHistory/LogsRenderer.tsx

```diff
@@ -36,8 +39,13 @@ export default function LogsRenderer({
         renderConfigurationError(isBlobStorageConfigured)
     ) : (
         <div className="logs__body">
-            {logs.map((log, index) => {
-                return <p className="mono fs-14" key={`logs-${index}`} dangerouslySetInnerHTML={createMarkup(log)} />
+            {logs.map((log: string, index: number) => {
+                return (
+                    <div className="flex left" key={`logs-${index}`}>
+                        <span className="cn-4 dc__mb-auto col-2 pr-10 mt-1">{index + 1}</span>
+                        <p className="col-10 mono fs-14 mh-24" dangerouslySetInnerHTML={createMarkup(log)} />
```

Comment:
also mh-24 & col-10 here

Reply:
```mh-24``` is necessary for empty logs, where without it, the row height is adapted to the line number height and breaking the symmetry.

Reply:
```col-10``` and ```col-2``` is used to divide the flex row into line number and output log. where I think col-10 can be skipped because it should natively adapt the width after col-2. I'll update that then 👍